### ISSUE TITLE
HF Model Card Example misses a `model.eval()` call.

### ISSUE THREAD

**Issue by PhilipMay** [oliverguhr/german-sentiment#5]
IMO on your example at the HF model card you miss a `model.eval()` call. From HF doc:

> - Set the model in evaluation mode to deactivate the DropOut modules
> - This is IMPORTANT to have reproducible results during evaluation!

See here: https://huggingface.co/oliverguhr/german-sentiment-bert#a-minimal-working-sample

What do you think?

**Comment by oliverguhr** [comment]
Right, that would be better 😬

**Comment by PhilipMay** [comment]
I tested it by the way. It makes no difference - which is strange. Well...

**Comment by oliverguhr** [comment]
Mh, well.. is it at least a bit faster?
